_includes/components/post-office-search.html

```diff
@@ -9,5 +9,8 @@
   {% assign uncached_translations_path = page.idp_translations_file | append: '?ts=' | append: timestamp %}
 
   <script src="{{uncached_translations_path | relative_url}}"></script>
+  <script type="text/javascript">
```

Comment:
Also, `type="text/javascript"` is unnecessary as the default, and inconsistent with the script we render below

```suggestion
  <script>
```